src/balatro_gym/cards/joker/joker.py

```diff
@@ -0,0 +1,303 @@
+            return 10
+        return 0
+
+
+class CrazyJoker(JokerBase):
+    _cost: int = 4
+
+    @property
+    def joker_type(self) -> Type:
+        return Type.ADDITIVE_MULT
+
+    @property
+    def rarity(self) -> Rarity:
+        return Rarity.COMMON
+
+    def get_mult(self, scored_cards: Sequence[PlayingCard], blind: BlindState, scored_hand: PokerHandType) -> int:
+        if (
+            scored_hand == PokerHandType.STRAIGHT
+            or scored_hand == PokerHandType.STRAIGHT_FLUSH
+            or scored_hand == PokerHandType.ROYAL_FLUSH
+        ):
+            return 12
+        return 0
+
+
+class DrollJoker(JokerBase):
+    _cost: int = 4
+
+    @property
+    def joker_type(self) -> Type:
+        return Type.ADDITIVE_MULT
+
+    @property
+    def rarity(self) -> Rarity:
+        return Rarity.COMMON
+
+    def get_mult(self, scored_cards: Sequence[PlayingCard], blind: BlindState, scored_hand: PokerHandType) -> int:
+        if (
+            scored_hand == PokerHandType.FLUSH
+            or scored_hand == PokerHandType.FLUSH_FIVE
+            or scored_hand == PokerHandType.FLUSH_HOUSE
+            or scored_hand == PokerHandType.STRAIGHT_FLUSH
+        ):
+            return 10
+        return 0
+
+
+class SlyJoker(JokerBase):
+    _cost: int = 3
+
+    @property
+    def joker_type(self) -> Type:
+        return Type.CHIPS
+
+    @property
+    def rarity(self) -> Rarity:
+        return Rarity.COMMON
+
+    def get_chips_hand(
+        self, scored_cards: Sequence[PlayingCard], blind: BlindState, board: BoardState, scored_hand: PokerHandType
+    ) -> int:
+        return 50 if get_num_pairs(scored_cards) >= 1 else 0
+
+
+class WilyJoker(JokerBase):
+    _cost: int = 4
+
+    @property
+    def joker_type(self) -> Type:
+        return Type.CHIPS
+
+    @property
+    def rarity(self) -> Rarity:
+        return Rarity.COMMON
+
+    def get_chips_hand(
+        self, scored_cards: Sequence[PlayingCard], blind: BlindState, board: BoardState, scored_hand: PokerHandType
+    ) -> int:
+        return 100 if contains_three_set(scored_cards) else 0
+
+
+class CleverJoker(JokerBase):
+    _cost: int = 4
+
+    @property
+    def joker_type(self) -> Type:
+        return Type.CHIPS
+
+    @property
+    def rarity(self) -> Rarity:
+        return Rarity.COMMON
+
+    def get_chips_hand(
+        self, scored_cards: Sequence[PlayingCard], blind: BlindState, board: BoardState, scored_hand: PokerHandType
+    ) -> int:
+        return 80 if get_num_pairs(scored_cards) >= 2 else 0
+
+
+class DeviousJoker(JokerBase):
+    _cost: int = 4
+
+    @property
+    def joker_type(self) -> Type:
+        return Type.CHIPS
+
+    @property
+    def rarity(self) -> Rarity:
+        return Rarity.COMMON
+
+    def get_chips_hand(
+        self, scored_cards: Sequence[PlayingCard], blind: BlindState, board: BoardState, scored_hand: PokerHandType
+    ) -> int:
+        return 100 if len(get_straight(scored_cards, board)) == 5 else 0
```

Comment:
here too we use a different funciton to retrieve the same info as Crazy joker (same for Crafty joker with Droll)